### PR TITLE
Notes Analyzer

### PR DESCRIPTION
**Pull request by MthreeJohnnyK**
I made a quick way to analyze notes.

### CODE REVIEW

**Review by geeknoid — APPROVED** [review]

File: app/src/main/java/com/example/myapplication/MatchData.java [hunk 1]
```diff
@@ -35,6 +35,9 @@ public enum Chain {
     Chain chaining = Chain.NOPE;
     int trapSucess = 0;
     int trapFail = 0;
+
+    double analScore = 0.0;
```
Comment (carlosfigueira) [inline app/src/main/java/com/example/myapplication/MatchData.java:39]: Suggestion: use full name `analyzerScore`. Same for method name on line 148

File: app/src/main/res/layout/fragment_second2.xml [hunk 2]
```diff
@@ -70,8 +70,16 @@
             android:text="Character Limit: 0/150"
             android:textSize="15dp"
             android:translationX="30dp"
-            android:translationY="608dp" />
-
+            android:translationY="580dp" />
+        <TextView
+            android:id="@+id/analScore"
```
Comment (carlosfigueira) [inline app/src/main/res/layout/fragment_second2.xml:75]: suggestion: don't abbreviate: `@+id/analyzerScore`

File: app/src/main/java/com/example/myapplication/Analyzer.java [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+				total += -10 * cur;
+				cur = 1.0;
+			} else if ((index = contains(multiWords, s)) >= 0) {
+				cur *= Double.parseDouble(multiWords[index].substring(0, multiWords[index].indexOf(",")));
+			}
+		}
+		return Math.round(total * 10)/10.0;
+	}
+	public static String keepLettersOnly(String str) {
+		String[] words = str.split("");
```
Comment (carlosfigueira) [inline app/src/main/java/com/example/myapplication/Analyzer.java:63]: Suggestion: this method can be implemented as a single line: `return str.replaceAll("[^a-z ]", "");`

Comment (vkirub) [reply app/src/main/java/com/example/myapplication/Analyzer.java:63]: should it include capitals letters?

Comment (MthreeJohnnyK) [reply app/src/main/java/com/example/myapplication/Analyzer.java]: @vkirub, all letters would be turned into lowercase before it goes through the function.

File: app/src/main/java/com/example/myapplication/MatchData.java (hunk 1 above)
Comment (MthreeJohnnyK) [reply app/src/main/java/com/example/myapplication/MatchData.java]: Done.

File: app/src/main/res/layout/fragment_second2.xml (hunk 2 above)
Comment (MthreeJohnnyK) [reply app/src/main/res/layout/fragment_second2.xml]: Done.

File: app/src/main/java/com/example/myapplication/Analyzer.java (hunk 3 above)
Comment (MthreeJohnnyK) [reply app/src/main/java/com/example/myapplication/Analyzer.java]: Done.

**Pull request merged**